mithril-test-lab/mithril-end-to-end/src/assertions/check.rs

```diff
@@ -257,9 +257,9 @@ pub async fn assert_node_producing_cardano_stake_distribution(
             Err(err) => Err(anyhow!(err).context(format!("Request to `{url}` failed"))),
         }
     }) {
-        AttemptResult::Ok(hash) => {
+        AttemptResult::Ok((hash, epoch)) => {
             info!("Aggregator produced a Cardano stake distribution"; "hash" => &hash);
```

Comment:
Since you have it now, adding the epoch to the log would be nice:
```suggestion
            info!("Aggregator produced a Cardano stake distribution"; "hash" => &hash, "epoch" => #?epoch);
```

Reply:
Done ✅